src/stdlib_hashmap_wrappers.f90

```diff
@@ -323,6 +340,21 @@ subroutine set_int8_key( key, value )
     end subroutine set_int8_key
 
 
+    subroutine set_int32_key( key, value )
+!! Version: Experimental
+!!
+!! Sets the contents of the key from an INTEGER(INT8) vector
```

Comment:
Should this comment have `INTEGER(INT32)`?

Reply:
This should be updated per latest revision.